servers/parser-graphql-wrapper/jest.setup.js

```diff
@@ -8,3 +8,4 @@ process.env.LETTER_INDEX =
   '[["a",0],["b",1],["c",2],["d",3],["e",4],["f",5],["0",6]]';
 process.env.SALT_1 = '123asdf';
 process.env.SALT_2 = 'asdaa47';
+process.env.PARSER_URL = 'http://example-parser.com/';
```

Comment:
In case your .env specifies the parser url, you need to make sure that  we always set it to the expected test value.